src/main/java/io/appium/java_client/MobileBy.java

```diff
@@ -69,6 +69,10 @@ protected String getLocatorString() {
      *
      * @param iOSAutomationText is iOS UIAutomation string
      * @return an instance of {@link io.appium.java_client.MobileBy.ByIosUIAutomation}
+     *
+     * @deprecated UIAutomation is going to get deprecated.
+     *        Use {@link io.appium.java_client.MobileBy.ByIosClassChain} instead
```

Comment:
there is no direct replacement for this locator. It is only available for the old UIA driver

Reply:
Sure. How about this 

* @deprecated UIAutomation is going to get deprecated.
*        It is recommended to use XCUITest